### PR TITLE
Hpe templates

### PR DESCRIPTION
**Pull request by dainok**
Implements the following templates for HPE Comware devices:

- display interface
- display ip vpn-instance
- display link-aggregation verbose

### CODE REVIEW

File: ntc_templates/templates/hp_comware_display_link-aggregation_verbose.textfsm [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Value LOADSHARING (.+)
+Value List LOCAL_INTERFACES (\S+)
+Value List REMOTE_INTERFACES (\S+)
+
+Start
+  ^Aggregat\S+\s+Interface:\s+${INTERFACE}
+  ^Creation\s+Mode:\s+${CREATION_MODE}
+  ^Aggregation\s+Mode:\s+${AGGREGATION_MODE}
+  ^Loadsharing\s+Type:\s+${LOADSHARING}
+  ^Local: -> LocalPorts
```
Comment (itdependsnetworks) [inline ntc_templates/templates/hp_comware_display_link-aggregation_verbose.textfsm:13]: Add ```  ^\s*$$
  ^. -> Error```

to all 3 states, (Start, LocalPorts, RemotePorts)

Comment (dainok) [reply ntc_templates/templates/hp_comware_display_link-aggregation_verbose.textfsm:13]: @itdependsnetworks fixed, thank you

File: ntc_templates/templates/hp_comware_display_interface.textfsm [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  ^\s{14,}(?:\s+[^,]+,){18}\s+${VLAN_PERMITTED},* -> Continue
+  ^\s{14,}(?:\s+[^,]+,){19}\s+${VLAN_PERMITTED},* -> Continue
+  ^\s{14,}(?:\s+[^,]+,){20}\s+${VLAN_PERMITTED},* -> Continue
+  # End od VLAN Passing
+  ^\s+VLAN\s+permitted:
+  ^\s{14,}
+  # Next
+  ^$$ -> Next.Record
+  # Drop
+  ^.*
```
[the remark this answers is not in the document]
Comment (itdependsnetworks) [reply ntc_templates/templates/hp_comware_display_interface.textfsm:130]: this should not have been resolved. `  ^.*` ensures that the lines below could never match and should be removed.

[the remark this answers is not in the document]
Comment (dainok) [reply ntc_templates/templates/hp_comware_display_interface.textfsm:130]: Thank you @itdependsnetworks. I have a question if you don't mind: I have a lot of useless output  (about interface errors) after VLAN info, that's why I drop all content after that. How should I approach?

[the remark this answers is not in the document]
Comment (itdependsnetworks) [reply ntc_templates/templates/hp_comware_display_interface.textfsm:130]: Just the old fashion tedious way of going through and accounting for each line. The problem is we have been burnt by this before and in order to maintain new templates this is just the standard that we require. Hopefully this https://ntc-templates.readthedocs.io/en/latest/user/faq/#why-is-there-a-requirement-to-use-error-in-every-template explains it well.

[the remark this answers is not in the document]
Comment (dainok) [reply ntc_templates/templates/hp_comware_display_interface.textfsm]: Thank you @itdependsnetworks for the advice. Template fixed (hopefully)

**Review by itdependsnetworks — APPROVED** [review]

**Review by itdependsnetworks — APPROVED** [review]

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

File: display lin-agg detailed.txt [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<SW01>disp link-a s[[ummary]]
```
[the remark this answers is not in the document]
Comment (dainok) [reply display lin-agg detailed.txt]: I'm so sorry, my fault. It was a tmp file used during my test., now deleted.

[the remark this answers is not in the document]
Comment (dainok) [reply display lin-agg detailed.txt]: @jmcgill298 I think it's ok now. I resolved the conversation but I still see "changes requested".

**Review by jmcgill298 — APPROVED** [review]

**Pull request merged**